.biobuddies/justfile

```diff
@@ -126,7 +126,7 @@ pc *args:
 # run Pre-Commit on All files
 pca *args:
     pre-commit {{ \
-        if `just cona` == 'helicopyter' { 'try-repo .' } else { 'run' } \
+        if `just cona` == 'helicopyter' { 'try-repo --config test.yaml .' } else { 'run' } \
```

Comment:
I haven't been able to get the config argument to really work. Maybe I can use Python to tweak the default `try-repo` configuration. The current blocker is changing the usual git repository for cookiecutter to the current directory.